app/main.py

```diff
@@ -91,12 +90,10 @@ def process(self, message, tx_id=None):
         try:
             self.publisher.publish_message(message, headers={'tx_id': tx_id})
         except PublishMessageError:
-            logger.exception('Unsuccessful publish. tx_id={}'.format(tx_id))
+            logger.exception('Unsuccessful publish.', tx_id=tx_id)
```

Comment:
Is it worth binding `tx_id` to the logger?

Reply:
I could, but it's only going to affect 2 lines.